rhombus/scribblings/ref-random.scrbl

```diff
@@ -0,0 +1,66 @@
+#lang scribble/rhombus/manual
+@(import:
+    "common.rhm" open:
+      except: def
+    "nonterminal.rhm" open
+    meta_label:
+      rhombus/random open)
+
+@title{Random Number Generation}
+
+@docmodule(rhombus/random)
+
+@doc(
+  annot.macro 'Random'
+  fun Random() :: Random
+  fun Random(state :: RandomState) :: Random
+){
+
+ Annotation and constructors for a pseudo-random number generator (PRNG).
+
+}
+
+@doc(
+  method (prng :: Random).random() :: Real.in(0 ~exclusive, 1 ~exclusive)
+  method (prng :: Random).random(n :: PosInt) :: Int.in(0, n ~exclusive)
+  method (prng :: Random).random(start :: Int, end :: Int)
+    :: Int.in(start, end ~exclusive)
+){
+
+ Steps @rhombus(prng) to obstain a number.
+
+ Using @rhombus(math.random) is the same as using
+ @rhombus(Random.current())'s @rhombus(Random.random).
```

Comment:
Was this meant to say `as using @rhombus(Random.current())'s @rhombus(random, ~datum) method`?

Reply:
HF